core/src/script/CGXP/api/Map.js

```diff
@@ -357,6 +394,12 @@ cgxp.api.Map.prototype = {
                 OpenLayers.Function.bind(this.recenter, this, center, zoom));
             return;
         }
+        center = this.autoProjection.tryProjection(center, this.map);
+        if (center == null){
+            center = []
+            center[0] = null;
+            center[1] = null;
+        }
         this.map.setCenter(new OpenLayers.LonLat(center[0], center[1]), zoom);
```

Comment:
I've observed that when the center components are null , it takes the center of the maxExtent, which is not bad. What do you think about it ?